software/contrib/cvecorder.py

```diff
@@ -86,7 +87,7 @@ def __init__(self):
 
         # Test routine, pick a random bank n times and save, then load the state
         if self.initTest:
-            #print(micropython.mem_info("level"))
+            print(micropython.mem_info("level"))
```

Comment:
Let's remove or comment this so we don't unnecessarily waste cycles when the modules isn't connected to a PC.

Reply:
This is behind the `initTest` flag, which needs to be manually enabled by editing the code. Presumable the user knows what they are doing if they are running this test.

Reply:
I missed that it was guarded by a flag. Thanks for clarifying. This addresses my concern.